hazelcast/src/main/java/com/hazelcast/query/impl/UUIDConverter.java

```diff
@@ -33,6 +33,9 @@ private UUIDConverter() {
 
     @Override
     public Comparable convert(Comparable value) {
+        if (value == null) {
```

Comment:
this was another converter where null-value handling was not implemented properly